imagetagger/imagetagger/images/models.py

```diff
@@ -43,7 +43,17 @@ class Meta:
     path = models.CharField(max_length=100, unique=True, null=True)
     name = models.CharField(max_length=100)
     location = models.CharField(max_length=100, null=True, blank=True)
-    description = models.TextField(max_length=1000, null=True, blank=True)
+    """
+    FOR SAFETY REASONS: use this description field to render html as seen below:
+    *************
+    import bleach
+    from bleach_whitelist import markdown_tags, markdown_attrs
+    import markdown
+
+    bleach.clean(markdown.markdown(imageset.description), markdown_tags, markdown_attrs))
```

Comment:
Use `mark_safe` here (or rather, in the yet-to-write model method) instead of treating the string as safe in the template. (see below)